lib/client/node_e2e_test.go

```diff
@@ -98,7 +98,7 @@ var _ = testutils.E2eDatastoreDescribe("Node tests", testutils.DatastoreEtcdV2,
 
 			// Apply node2 with spec1.
 			By("Applying node2 with spec1")
-			_, err = c.Nodes().Update(&api.Node{Metadata: meta2, Spec: spec1})
+			_, err = c.Nodes().Apply(&api.Node{Metadata: meta2, Spec: spec1})
```

Comment:
Why does this need to be apply now?

Reply:
Thought I may have caught an error, the By line states `By("Applying node2 with spec1")` and we were calling update, I can revert it if this was intentional, but perhaps the comment should be updated?

Reply:
Yep I think you might be right.